reflex/.templates/apps/basetemplate/code/sidebar.py

```diff
@@ -0,0 +1,124 @@
+import reflex as rx
+from .state import State
+from .styles import *
+
+
+def sidebar_item(text: str, icon: str, url: str) -> rx.Component:
+    return rx.hstack(
+        rx.image(
+            src=icon,
+            height="2.5em",
+            padding="0.5em",
+        ),
+        rx.text(
+            text,
+        ),
+        bg=rx.cond(State.current_page == text, accent_color, "transparent"),
+        color=rx.cond(State.current_page == text, accent_text_color, text_color),
+        border_radius=border_radius,
+        border="1px solid transparent",
+        width="100%",
+        padding_x="1em",
+        _hover={
+            "box_shadow": box_shadow,
+        },
+        on_click=lambda: State.set_page(text),
+    )
+
+
+def sidebar_header():
+    return rx.hstack(
+        rx.image(
+            src="/icon.svg",
+            height="2em",
+        ),
+        rx.spacer(),
+        rx.link(
+            rx.center(
+                rx.image(
+                    src="/github.svg",
+                    height="3em",
+                    padding="0.5em",
+                ),
+                box_shadow=box_shadow,
+                bg="transparent",
+                border_radius=border_radius,
+                _hover={
+                    "bg": "#F5EFFE",
+                },
+            ),
+            href="https://github.com/reflex-dev/reflex",
+        ),
+        width="100%",
+        border_bottom=border,
+        padding="1em",
+    )
+
+
+def sidebar_footer():
+    return rx.hstack(
+        rx.link(
+            rx.center(
+                rx.image(
+                    src="/paneleft.svg",
+                    height="2em",
+                    padding="0.5em",
+                ),
+                bg="transparent",
+                border_radius=border_radius,
+                on_click=State.toggle_sidebar,
+                _hover={
+                    "bg": "#F5EFFE",
+                },
+            ),
+            href="https://github.com/reflex-dev/reflex",
```

Comment:
```suggestion
```

This causes the sidebar to not collapse, but instead navigate away

Reply:
Yes I got rid the the collapsable sidebar for simplicity reasons. I need to get rid of the toggle_sidebar, show_sidebar code will do that now